### PR TITLE
Refactor backend side

### PR DESCRIPTION
**Pull request by henrikra**
Mainly refactoring `index.ts` shorter. By refactoring I learn easier and faster the whole code base :) Also I have formatted some files with prettier so everything is consistent 🎉 

I guess you are previously Java coder right @dolanmiu ? ;)

### CODE REVIEW

**Comment by dolanmiu** [comment]
Yes previously Java, then C#, and now Angular/Node... wait I am a "full-stack" dev at my current job actually, so I do Java lol

How did you know?

File: src/main/index.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ class Main {
       clientSecret: uncheckedConfig.clientSecret,
       deviceId: uncheckedConfig.deviceId,
       refreshToken: uncheckedConfig.refreshToken,
-      lite: uncheckedConfig.lite ? uncheckedConfig.lite : false,
+      lite: uncheckedConfig.lite || false,
```
Comment (dolanmiu) [inline src/main/index.ts]: This is good, I keep forgetting this feature!

**Comment by henrikra** [comment]
I am just guessing from your code style :P Lots of classes. That usually means Java background. I more like a functional programmer. Have you done any of that?

**Comment by dolanmiu** [comment]
Can we use 4 spaces? I prefer this over 2

**Comment by henrikra** [comment]
I have set prettier to 4 spaces now. I am more like 2 spaces guys since it is more compact 😎

**Comment by dolanmiu** [comment]
Ah yeah, I like splitting up code as much as possible, less to focus on, and easier to test. Perhaps it came from my Java background.

I never understand why developers create 700 line files. 

And yeah I've done functional programming, my newer stuff is quite functional. I tend to do a hybrid OOP + functional these days.

I have wrote some horrendus long functional snippets, when I got too excited about functional 😂

**Comment by dolanmiu** [comment]
Everyone at my work likes 2 spaces, and the internet seems to like 2 spaces too, but I find it harder to read cos everything is squashed LOL! I think I am alone on this one

**Comment by henrikra** [comment]
Yeah people are different :P

**Pull request merged**